### ISSUE TITLE
[Feature Request] PaddlePaddle model quantization support

### ISSUE THREAD

**Issue by xaclincoln** [openvinotoolkit/openvino#11802]
Hi, thanks for the excellent job ! 
NNCF only support Tensorflow and pytorch at present. Is paddlepaddle support on the plan?
PaddlePaddle indeed has a native quantization framework named PaddleSlim, but the quantized models are not covertible to onnx and IR.  Or could model optimizer support converting quantized paddlepaddle models?
What I want to achieve is inference a quantized paddle paddle model using openvino, as openvino is more optimized than other framework on intel hardware ( in production inference environment,  an intel AI edge box is used).

**Comment by dlod-openvino** [comment]
Add one!

**Comment by dlod-openvino** [comment]
Agree! When NNCF can support PaddlePaddle?

**Comment by wenjiew** [comment]
Paddle quantized model is not supported in OpenVINO yet. Model optimizer cannot convert quantized models from PaddleSlim.
One alternative approach is:
1) Use model optimizer to convert original PaddlePaddle model to OpenVINO IR.
2) Use OpenVINO POT or NNCF tool to quantize the converted IR.

**Comment by xaclincoln** [comment]
Hi, [as the docs tells](http://www.uzelinfo.com/page118?product_category=1&brd=1),  NNCF requires tensorflow2 or pytorch based models to perform quantization-awared training, could you provide any doc about quantize IR models using NNCF?

**Comment by wenjiew** [comment]
Here is documentation on NNCF usage: https://github.com/openvinotoolkit/nncf/blob/develop/docs/Usage.md
There is also this tutorial on how to use POT: https://docs.openvino.ai/latest/pot_default_quantization_usage.html

**Comment by dlod-openvino** [comment]
> Paddle quantized model is not supported in OpenVINO yet. Model optimizer cannot convert quantized models from PaddleSlim. One alternative approach is:
> 
> 1. Use model optimizer to convert original PaddlePaddle model to OpenVINO IR.
> 2. Use OpenVINO POT or NNCF tool to quantize the converted IR.

When OpenVINO can support the Paddle quantized model from PaddleSlim? PaddleSlim release a super easy-to-use tool: [Auto Compression Toolkit](https://github.com/PaddlePaddle/PaddleSlim/tree/develop/example/auto_compression), Both TensorRT and ONNX runtime have already support it.
![TensorRT support PaddleSlim](https://user-images.githubusercontent.com/59691336/185848103-b8d19d52-ed7d-477f-bdb3-8fef4d48ee34.png)

**Comment by jgespino** [comment]
Ref. 69461

**Comment by xczhai** [comment]
We have already supported the native PaddleSlim model in OpenVINO PDPD FE!

- support the quantized PDPD format model loading.
- enable the quantize_linear and dequantize_linear op.
- enable the FQ fusion.
- keep the PDPD FE op test coverage.
- verify the accuracy and performance.

For more details, please see [- PaddlePaddle model quantization support](https://github.com/openvinotoolkit/openvino/pull/14834)

**Comment by ceciliapeng2011** [comment]
> We have already supported the native PaddleSlim model in OpenVINO PDPD FE!
> 
> * support the quantized PDPD format model loading.
> * enable the quantize_linear and dequantize_linear op.
> * enable the FQ fusion.
> * keep the PDPD FE op test coverage.
> * verify the accuracy and performance.
> 
> For more details, please see [- PaddlePaddle model quantization support](https://github.com/openvinotoolkit/openvino/pull/14834)

@xaclincoln Has it meet your requirements? Could you please have a try with openvino master branch? Looking forward to your feedback! Thanks!

**Comment by hbalasu1** [comment]
Close this Case since PR #14834 has been merged.
